src/Nbobtc/Bitcoind/Client.php

```diff
@@ -19,13 +19,25 @@ class Client implements ClientInterface
     protected $cacert;
 
     /**
-     * @param string|null $dsn
-     * @param string|null $cert The file path to a certificate you'd like to use for SSL verification
+     * @var bool
      */
-    public function __construct($dsn = null, $cacert = null)
+    protected $keepAlive;
+
+    /**
+     * @var resource|null
+     */
+    private $ch = null;
+
+    /**
+     * @param string|null   $dsn
+     * @param string|null   $cacert     The file path to a certificate you'd like to use for SSL verification
+     * @param bool          $keepAlive
+     */
+    public function __construct($dsn = null, $cacert = null, $keepAlive = false)
```

Comment:
personally I'd say it should default to TRUE, but by defaulting to FALSE it means there's absolutely no BC breaks, though I doubt this could break anything.